depthmapX/depthmapView.h

```diff
@@ -178,6 +178,7 @@ public slots:
    void ResetHoverWnd(const QPoint& p = QPoint(-1,-1));
    void CreateHoverWnd();
 
+   void ZoomTowards(double ratio, const Point2f& point);

```

Comment:
What's the diff between ZoomIn and ZoomTowards? Needs at least to be explained if we need both functions.